tests/mock_robotd.py

```diff
@@ -9,6 +12,20 @@
 from robotd.master import BoardRunner
 
 
+def create_root_dir():
+    return tempfile.mkdtemp(prefix="robot-api-test-")
+
+
+def remove_root_dir(dir):
```

Comment:
Can we please avoid shadowing a builtin here: `dir` -> `directory`.

Reply:
No longer necessary as this function is removed by 8275d72.